pagopa/io-app
# fix: [IOBP-593] Fix back button in PSP selection screen during payment

## Short description
This PR fixes the back button in the PSP selection screen during a payment.

## List of changes proposed in this pull request
- `handleGoBack` function refactoring

## How to test
Using the `io-dev-api-server`, start a new payment. Check that you can go back during the PSP selection step.
